templates/kubernetes/what-is-kubernetes.html

```diff
@@ -284,226 +305,339 @@ <h2 class="p-muted-heading u-align--center">
+  <div class="row">
+    <div class="col-12">
       <h2>
         How does Kubernetes work?
       </h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-5">
       <p>
-        Kubernetes works by joining a group of physical or virtual host machines referred to as "nodes", into a cluster to manage containers. This creates a "supercomputer" that has greater processing speed, more storage capacity and increased network capabilities than any single machine would have on its own. The nodes include all necessary services to run "pods", which in turn run single or multiple containers. A pod corresponds to a single instance of an application in Kubernetes.
+        Kubernetes works by joining a group of physical or virtual host machines, referred to as "nodes", into a cluster. This creates a "supercomputer" to run containerized applications with a greater processing speed, more storage capacity, and increased network capabilities than any single machine would have on its own. The nodes include all necessary services to run "pods", which in turn run single or multiple containers. A pod corresponds to a single instance of an application in Kubernetes.
       </p>
       <p>
-        One (or more for larger clusters, or High Availability) node of the cluster is designated as the "control plane". The control plane node then assumes responsibility for the cluster as the orchestration layer - scheduling and allocating tasks to the other "worker" nodes in a way which maximises the resources of the cluster. All operator interaction with the cluster goes through this main node, whether that is changes to the configuration, executing or terminating workloads, or controlling ingress and egress on the network.
+        One (or more for larger clusters, or High Availability) node of the cluster is designated as the "control plane". The control plane node then assumes responsibility for the cluster as the orchestration layer &ndash; scheduling and allocating tasks to the "worker" nodes in a way which optimises the resources of the cluster. All administrative and operational tasks on the cluster are done through the control plane, whether these are changes to the configuration, executing or terminating workloads, or controlling ingress and egress on the network.
       </p>
       <p>
         The control plane is also responsible for monitoring all aspects of the cluster, enabling it to perform additional useful functions such as automatically reallocating workloads in case of failure, scaling up tasks which need more resources and otherwise ensuring that the assigned workloads are always operating correctly.
       </p>
       <p>
         <a href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Download Enterprise Kubernetes datasheet&nbsp;&rsaquo;</a>
       </p>
-      <p>
-        <img class="u-hide--small" src="https://assets.ubuntu.com/v1/fa6db665-K8s_Architecture_updated.svg" alt="">
-      </p>
     </div>
```

Comment:
Wondering if the image needs to be below the text following this [design](https://miro.com/app/board/o9J_ln7Z8gk=/)?

Reply:
Only the 'did you know' and glossary sections were updated in the design file ([relevant comment here](https://github.com/canonical-web-and-design/web-squad/issues/4551#issuecomment-953074003)), so I just followed the copy doc for this image placement. @spencerbygraves let me know if you'd prefer it underneath instead!